lib/dig.py

```diff
@@ -12,46 +12,51 @@
         'minecraft:lapis_lazuli':True,
 }
 
+navigator = nav.Navigator()
```

Comment:
Would we want this namespaced to quarry?

With this being global, I can imagine some very confusing cases where its state is modified by other functions